ui/components/component-library/text/text.scss

```diff
@@ -59,6 +59,11 @@ $text-variants: (
       @else {
         font-weight: $weight;
       }
+
+      // Rule for strong tag inside Text component
+      strong {
+        font-weight: var(--font-weight-bold);
+      }
```

Comment:
I believe there has been a PR merged for this issue. I think this can be removed otherwise it will be duplicate code
https://github.com/MetaMask/metamask-extension/pull/18816

```suggestion
```

Reply:
Updated in https://github.com/MetaMask/metamask-extension/pull/18752/commits/bbe260718b04bf05c0074823520d157bb12a4938